libdcadec/idct_fixed.c

```diff
@@ -28,11 +28,9 @@ static void sum_a(const int * restrict input, int * restrict output, int len)
 
 static void sum_b(const int * restrict input, int * restrict output, int len)
 {
-    for (int i = 0; i < len; i++) {
-        if (i > 0)
-            output[i] = input[2 * i] + input[2 * i - 1];
-        else
-            output[i] = input[2 * i];
+    output[0] = input[0];
+    for (int i = 1; i < len; i++) {
```

Comment:
If I may say something. Braces should be removed if we want to keep the style consistent.